### PR TITLE
mock data for my team page challenge

### PR DESCRIPTION
**Pull request by susird**
This is the mock data for My team page :smile_cat:

### CODE REVIEW

File: data.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+export const data = {
+  "header": {
+    "title-team": "The creative crew",
+    "description-team-title": "Who we are",
+    "description-team-text": "We are team of creatively diverse. driven. innovative individuals working in various locations from the world."
+  },
+
```
Comment (josepplloo) [inline data.js]: You can delete these blank lines if you want

File: data.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  "team-member-section": {
+    "team-member-card": {
+      "image": "/home/susanar/Documentos/Programacion_2021_1/Layouts/my-team-page/photo4.png",
+      "name-title": "Skylah Lu",
+      "charge-text": "UX DESIGNER"
+    }
+  },
+
+  "footer": {
+    "socialMedia":
```
Comment (josepplloo) [inline data.js]: If you're using kebab case instead camel case this would be social-media

File: data.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      [
+        {
+          "href": "https://instagram.com/ve5pula",
+          "image": "https://www.flaticon.es/svg/static/icons/svg/725/725339.svg"
```
Comment (josepplloo) [inline data.js]: Insert aria for images: alt or an id

**Review by josepplloo — CHANGES_REQUESTED** [review]
Try to be more consistent

File: data.js (hunk 1 above)
Comment (susird) [reply data.js]: fac9e7d

File: data.js (hunk 2 above)
Comment (susird) [reply data.js]: fac9e7d

File: data.js (hunk 3 above)
Comment (susird) [reply data.js]: fac9e7d

File: data.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
+export const data = {
+  "header": {
+    "title-team": "The creative crew",
+    "description-team-title": "Who we are",
+    "description-team-text": "We are team of creatively diverse. driven. innovative individuals working in various locations from the world."
```
Comment (anyruizd) [inline data.js]: I know this text is according to the designs, but it makes more sense like this: 
```suggestion
    "description-team-text": "We are a team of creatively diverse-driven innovative individuals working in various locations from the world."
```

File: data.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+          "alt": "Instagram Image"
+        },
+        {
+          "href": "https://twitter.com/ve5pula",
+          "image": "https://www.flaticon.es/svg/static/icons/svg/725/725372.svg",
+          "alt": "Twitter Image"
+        }
+      ]
+  }
+}
```
Comment (anyruizd) [inline data.js]: I think you need to repeat this data as many times as cards are on the page or create different cards to populate the team page properly.

Comment (susird) [reply data.js]: 10e33b8

File: data.js (hunk 4 above)
Comment (susird) [reply data.js]: 10e33b8

**Review by anyruizd — APPROVED** [review]

File: data.js [hunk 6]
```diff
@@ -0,0 +1,56 @@
+export const data = {
+  "header": {
+    "title-team": "The creative crew",
+    "description-team-title": "Who we are",
+    "description-team-text": "We are a team of creatively diverse-driven innovative individuals working in various locations from the world."
+  },
+  "team-member-section": {
+    "team-member-card": [
+      {
+        "image": "/home/susanar/Documentos/Programacion_2021_1/Layouts/my-team-page/photo1.png",
```
Comment (josepplloo) [inline data.js]: use a relative path here

**Review by josepplloo — CHANGES_REQUESTED** [review]

**Review by josepplloo — APPROVED** [review]

**Pull request merged**